IamCredentials/composer.json

```diff
@@ -23,8 +23,7 @@
         }
     },
     "require": {
-        "google/gax": "^1.7.0",
-        "google/common-protos": "^1.3.1||^2.0"
```

Comment:
@bshaffer For IamCredentials/ServiceControl/Servicemangement/main composer.json, this dependency on 1.3.1 or higher was done on purpose - can we please leave? :)

Reply:
That makes sense! I've added them back.

Reply:
@dwsupplee if we set `google/gax` to have a minimum of `^1.3.1` for `google/common-protos`, we could remove these extra requires. WDYT?

Reply:
A user could have an older version of GAX installed and fall through the cracks, so I prefer we keep this approach for now. We can just drop 1 & 2 completely come our major version bumps if that sounds good to you.